### ISSUE TITLE
About color inconsistency 

### ISSUE THREAD

**Issue by Lsiming** [JiahuiYu/generative_inpainting#104]
There are many color inconsistencies in my results.  Do you have any good Suggestions? Should other loss functions be added

**Comment by Lsiming** [comment]
I repair RGB images, the result is not blur, but the color is slightly different from the surrounding. Of course, I fixed it on the irregular mask. Have you considered adding the loss function of punishment color in the experiment?
![example](https://user-images.githubusercontent.com/38555268/43430799-5463c124-949d-11e8-9501-9ae55758d1c6.png)

**Comment by Irvingwangjr** [comment]
sss

**Comment by JiahuiYu** [comment]
Do you use free-form masks in training?
